ansible_ai_connect/users/reports/postman.py

```diff
@@ -218,7 +219,8 @@ def get_drive(self) -> GoogleDrive:
     def create_filename(title: str, report_date: datetime):
         if report_date.tzinfo is None:
             raise ValueError
-        return f"{report_date.strftime('%Y%m%d')}_{title}.csv"
+        day_before = report_date - relativedelta(days=1)
```

Comment:
`report_date` is *created_before*

See https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/users/reports/postman.py#L226

Therefore if you really want "day before `created_after`.." then L#226 should also be changed accordingly.